aspnetcore/index.md

```diff
@@ -16,66 +16,47 @@ uid: index
 
-## Why build ASP.NET Core?
+## Why use ASP.NET Core?
 
-The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web apps, and over the years we have added and evolved many capabilities to it.
+ASP.NET released over 15 years ago.  Millions of developers used it to create web apps. Each new release added features.
 
-ASP.NET Core has a number of architectural changes that result in a much leaner and modular framework.  ASP.NET Core is no longer based on *System.Web.dll*. It is based on a set of granular and well factored [NuGet](http://www.nuget.org/) packages. This allows you to optimize your app to include just the NuGet packages you need. The benefits of a smaller app surface area include tighter security, reduced servicing, improved performance, and decreased costs in a pay-for-what-you-use model.
+ASP.NET Core has a number of architectural changes resulting in a leaner and modular framework.  ASP.NET Core is no longer based on *System.Web.dll*. It is based on a set of granular and well factored [NuGet](http://www.nuget.org/) packages. This allows you to optimize your app to include just the NuGet packages you need. The benefits of a smaller app surface area include tighter security, reduced servicing, improved performance, and <!-- How does it decrease costs? --> decreased costs in a <!-- How does pay for use work? --> *pay for what you use* model.
 
-With ASP.NET Core you gain the following foundational improvements:
-
-* A unified story for building web UI and web APIs
-
-* Integration of [modern client-side frameworks](client-side/index.md) and development workflows
-
-* A cloud-ready environment-based [configuration system](fundamentals/configuration.md)
-
-* Built-in [dependency injection](fundamentals/dependency-injection.md)
-
-* New light-weight and modular HTTP request pipeline
-
-* Ability to host on IIS or self-host in your own process
-
-* Built on [.NET Core](https://microsoft.com/net/core), which supports true side-by-side app versioning
-
-* Ships entirely as [NuGet](https://nuget.org)  packages
-
-* New tooling that simplifies modern web development
-
-* Build and run cross-platform ASP.NET apps on Windows, Mac and Linux
+ASP.NET Core provides the following foundational improvements:
```

Comment:
Looks like there are 2 spaces before the 2nd sentence. Remove 1 of them. Also recommend adding a comma after "architectural changes" and hyphenating "well factored".